### ISSUE TITLE
On clicking study statistics on the Nav Bar , the user is getting redirected to "503 : Service is currently unavailable " page.

### ISSUE THREAD

**Issue by Dibyajyoti2002** [narayan954/Acadmica#108]
![image](https://user-images.githubusercontent.com/72976359/227519840-a1c0b846-7623-42ee-b41a-39e61d0562a6.png)

**Comment by narayan954** [comment]
> ![image](https://user-images.githubusercontent.com/72976359/227519840-a1c0b846-7623-42ee-b41a-39e61d0562a6.png)

Yeah, that's because the page has not yet been implemented and is a WIP, It'll need a separate issue to resolve. Thanks for pointing that out
